src/archival/utils.rs

```diff
@@ -59,6 +79,51 @@ pub async fn is_row_exists(pool: &PgPool, row_id: i32) -> bool {
     }
 }
 
+pub async fn make_archival_network_request(
+    url: &str,
+) -> Result<String, Box<dyn std::error::Error + Send + Sync>> {
+    let endpoint_url = "https://web.archive.org/save";
+    let mut headers = header::HeaderMap::new();
+    headers.insert("Accept", "application/json".parse().unwrap());
+    headers.insert(
+        "Authorization",
+        "LOW iJN8ly6eMroQjKfd:TxLzPGdXKMWvLLuY".parse().unwrap(),
```

Comment:
I think you may have committed an api key

Reply:
Yeah, kinda intentional. I didn't want to put env (for now). Since these credentials are linked with my personal account, they don't mean much. Will use env when I get MeB's credentials.

Reply:
Just discussed in the chat: Using the crate `config` should be a better fit.

Don’t forget to change your key then.